repo-scripts/size-analysis/analysis-helper.ts

```diff
@@ -348,8 +387,12 @@ function handleExportStatementsWithFromClause(
     node.exportClause.elements.forEach(exportSpecifier => {
       declarations.variables.push(exportSpecifier.name.escapedText.toString());
     });
-  } else {
-    console.log(`${moduleName}: ${Warning.MODULE_NOT_RESOLVED}`);
+  }
+  // handles the case when exporting * from a module whose location can't be resolved
+  else {
+    console.log(
+      `export statement: export * from ${moduleName}: ${Warning.MODULE_NOT_RESOLVED}`
```

Comment:
Can you add some context in the message? something like, `the public API of moduleA is not complete because it reexports from 'moduleName' using * export, but we couldn't resolve 'moduleName' `